src/Experimental/isomorphism.jl

```diff
@@ -46,7 +61,7 @@ Return `true` if the graph `g1` contains a vertex induced subgraph that is isomo
 - `edge_relation`: A binary function that takes an edge from `g1` and one from `g2`. An
     isomorphism only exists if this function returns `true` for all matched edges.
 - `alg`: The algorithm that is used to find the induced subgraph isomorphism. Can be only
-    `:vf2` at the moment.
+    `VF2()` at the moment.
```

Comment:
`alg=VF2()` if typed manually will only work with `using LightGraphs.Experimental`, otherwise you need to specify it exactly, i.e, `LightGraphs.Experimental.VF2()` . Do I need mention that in the help somehow?

It's quite different to the symbol which didn't need that at all.